shapes/profile/app.shex

```diff
@@ -0,0 +1,36 @@
+PREFIX oidc: <http://www.w3.org/ns/solid/oidc#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schem: <http://schema.org/>
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX acl: <http://www.w3.org/ns/auth/acl#>
+PREFIX cert:  <http://www.w3.org/ns/auth/cert#>
+PREFIX ldp: <http://www.w3.org/ns/ldp#>
+PREFIX sp: <http://www.w3.org/ns/pim/space#>
+PREFIX solid: <http://www.w3.org/ns/solid/terms#>
+
+<#id> {
+  a schem:SoftwareApplication
+    // rdfs:comment  "A software application. (from Schema.org)" ;
+  oidc:redirect_uris IRI + 
+    // rdfs:comment  "URIs where the Solid Identity Provider may redirect the user to complete the login process" ;
+  oidc:scope xsd:string ?
```

Comment:
FYI, this field is not part of the Solid-OIDC specification. I'd remove this.

Reply:
Which one? I found both `redurect_uris` and `scope` here: https://github.com/solid/vocab/blob/main/solid-oidc.ttl

Reply:
Interesting. I was referring to `scope`
It certainly won't hurt to include that here